analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/FileMetadataAnalyzerTest.cs

```diff
@@ -76,17 +76,15 @@ public void NotAutogenerated(ProjectType projectType)
             CreateBuilder(projectType, notAutogeneratedFiles)
                 .WithSonarProjectConfigPath(TestHelper.CreateSonarProjectConfig(TestContext, projectType))
                 .VerifyUtilityAnalyzer<FileMetadataInfo>(messages =>
+                    messages.Should().BeEquivalentTo(notAutogeneratedFiles.Select(expected => new FileMetadataInfo
```

Comment:
Isnt' this going to generate a message that is very hard to read and compare?

Reply:
There isn't much we can do about this. The files are no longer sorted in a stable way. `BeEquivalentTo` and `SatisfyRespectively` are the best assertion methods we have for such a scenario. I assume the error messages are stating the missing/additional entries in a sane manner.